.travis.yml

```diff
@@ -61,10 +61,21 @@ matrix:
           packages:
             - gcc-multilib
             - libgmp-dev:i386
+    - compiler: gcc
+      env:
+        - FIELD=auto  BIGNUM=no  SCALAR=auto  ENDOMORPHISM=yes  STATICPRECOMPUTATION=yes  ASM=x86_64
+        - EXPERIMENTAL=no ECDH=no  RECOVERY=no VALGRIND=yes EXTRAFLAGS="--disable-openssl-tests CPPFLAGS=-DVALGRIND"  HOST= BUILD= JNI=no
```

Comment:
I think we want ECDH and RECOVERY enabled here?

Reply:
On my first tries it took ~25 minutes, but now with `./test 8` it might be faster, i'll try re-enabling them.

Reply:
It worked in my testing without endo and with schnorrsig. Without enabling all features we would still want the memsanitizer PR.

Reply:
see https://travis-ci.org/jonasnick/secp256k1/jobs/609452969